data-pipeline/src/pipeline/input_schemas.py

```diff
@@ -321,11 +300,13 @@
     "BNCH21606 (Agency supply teaching staff)": "float",
     "BNCH21403 (Energy)": "float",
     "BNCH21402 (Water and sewerage)": "float",
+    "Valid To": "string",
```

Comment:
Although this has a `Valid From` counterpart, as per the `academy_master_list` above, it doesn't appear to be used anywhere.

Reply:
Probably something we can pick up with Ben to ensure we've defined the required input fields mappings